cmd/stackdriver-prometheus-sidecar/statusz.go

```diff
@@ -54,22 +53,17 @@ func (h *statuszHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 			ClusterLocation string
 			ClusterName     string
 		}
-		ConfigTable    map[string]string
-		MetricRenames  map[string]string
-		StaticMetadata []scrape.MetricMetadata
+		Config *mainConfig
 	}
-
 	data.ServerName = filepath.Base(os.Args[0])
 
 	data.VersionInfo = version.Info()
 	data.BuildContext = version.BuildContext()
 	data.Uname = Uname()
 	data.FdLimits = FdLimits()
 
-	data.StartTime = serverStart.Format(time.RFC1123)
-	uptime := int64(time.Since(serverStart).Seconds())
-	data.Uptime = fmt.Sprintf("%d hr %02d min %02d sec",
-		uptime/3600, (uptime/60)%60, uptime%60)
+	data.StartTime = serverStart.Format(time.RFC3339)
+	data.Uptime = time.Since(serverStart).String()
```

Comment:
Nit, I'm _guessing_ the template system will happily accept the time and duration objects, and may even provider better formatting options. It's fine if you decide to keep this as-is.

Reply:
Done.